lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2116,6 +2116,17 @@ namespace {
           continue;
         }
 
+        if (auto friendDecl = dyn_cast<clang::FriendDecl>(m)) {
+            auto d = VisitFriendDecl(friendDecl);
+
+            // Check if the friend decl was converted into a FuncDecl
```

Comment:
Should probably remove this and add doc-string to `VisitFriendDecl` saying that this function returns the Swift imported version of the "friend" (nested) decl found inside this `clang::FriendDecl`